docs/usage/configuration-options.md

```diff
@@ -706,22 +706,25 @@ You can define custom managers to handle:
 - Proprietary file formats or conventions
 - Popular file formats not yet supported as a manager by Renovate
 
-Currently we only have one custom manager.
-The `regex` manager which is based on using Regular Expression named capture groups.
+Renovate has two custom managers:
 
-You must have a named capture group matching (e.g. `(?<depName>.*)`) _or_ configure its corresponding template (e.g. `depNameTemplate`) for these fields:
+| Custom manager | Matching engine                                |
+| -------------- | ---------------------------------------------- |
+| `regex`        | Regular Expression, with named capture groups. |
+| `jsonata`      | JSONata query.                                 |
+
+You must capture/extract the following three fields _or_ configure its corresponding template (e.g. `depNameTemplate`) for these fields:
 
 - `datasource`
 - `depName` and / or `packageName`
 - `currentValue`
 
-Use named capture group matching _or_ set a corresponding template.
 We recommend you use only _one_ of these methods, or you'll get confused.
 
-We recommend that you also tell Renovate what `versioning` to use.
+We also recommend that you also tell Renovate what `versioning` to use.
 If the `versioning` field is missing, then Renovate defaults to using `semver` versioning.
```

Comment:
Copying `@viceice`'s comment:

> this isn't fully true. the datasource can set a different versioning (most data sources doing it). we should suggest this only if the datasource default versioning isn't suitable

What do you think of this text?

```suggestion
Renovate defaults to `semver` versioning if _both_ condition are met:

- The `versioning` field is missing in the custom manager config
- The Renovate datasource does _not_ set its own default versioning
```

Reply:
I need to check, but it probably uses `semver-coerced` as fallback 🤔

Reply:
Yes, it is `semver-coerced`.

Reply:
I updated my suggestion so it says `semver-coerced`. 😉